index.html

```diff
@@ -212,7 +212,44 @@ <h1>
         </section>
       </section>
     </ul>
-  </section> 
   </section>
+  </section>
+
+  <!-- Form section of Portfolio -->
+  <section class="form_s">
+    <h1>
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h1>
+    <form class="form" action="https://formspree.io/f/xlekzalj" method="POST">
+      <fieldset>
+        <ul>
+          <li><input type="text" name="user_name" placeholder="Full name" required ></li>
+          <li><input type="email" name="user_mail" placeholder="Email address" required ></li>
+          <li><textarea name="user_msg" id="msg" cols="30" rows="10" required>Hey I'd like to connect to...</textarea></li>
```

Comment:
- [ ] Please from the project requirement, the textarea field should accept up to 500 characters so I think you should add the `maxlength` attribute for the textarea field and make it accept 500 chars :pray:

Reply:
Ok done